webpack.prod.config.js

```diff
@@ -60,7 +60,7 @@ module.exports = merge(baseConfig, {
     extractSass,
     new webpack.DefinePlugin({
       'process.env': {
-        'NODE_ENV': JSON.stringify("production"),
+        'NODE_ENV': JSON.stringify(process.env.NODE_ENV || 'development'),
```

Comment:
development? that seems wrong, should be production no?

Reply:
yep, I'm just testing it to see if it hides logging correctly etc... 
Please don't merge yet